### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Datagirlcmr**
This is the corrected version from the first pull request . The project is a clone of the Youtube Landing Page

### CODE REVIEW

**Comment by shloch** [comment]
Hi @Selma and @Tresor11  👋,

You've done a great job setting up this project so far. 👍👍
Few changes required though, then we're good to go

- Validate your HTML markup (you can use --> https://validator.w3.org/ ). Almost valid, one more thing lacking, please check it out

- Remove inline styling, use only your css for styling

_**Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you.

_`When re-submitting for code review, please dont delete this Pull Request, just do your modifications and fill code review form with this same Pull request link`_

Happy coding 😄,
--SHEY**_

**Comment by Tresor11** [comment]
we've removed the inline styling and also the code is passing the validation without any error or warning

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    <script src="https://kit.fontawesome.com/cce6e1325c.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <header>
+        <nav>
+            <div id="nav">
+                <label>
+                <img src="/image/icons8-menu-26.png" alt="menu"> 
+                <img src="/image/logo.png" id="logo" alt="logo">
```
Comment (Redvanisation) [inline index.html]: The icons are not showing, to fix it you guys can put `./` at the beginning of the image path like this `src="./image/icons8-menu-26.png"`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+               </label>
+
+                <form>
+                <input type="search" placeholder="Search" size="70">
+            <button>
+                <img src="/image/icons8-search-26.png" alt="search">
```
Comment (Redvanisation) [inline index.html]: Same issue as the previous icons

File: index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            </button>
+        </form>
+
+        <label id="left"><img src="/image/icons8-documentary-26.png" alt="video">
+            <img src="/image/icons8-list-26.png" alt="list">
+            <img src="/image/icons8-notification-26.png" alt="notif">
```
Comment (Redvanisation) [inline index.html]: And the same issue here too

File: index.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+                    <iframe src="https://www.youtube.com/embed/jeJRIk1hd4E" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+                <div id="info">
+                    <div id="views">
+                        <h3>Contributing to Open Source Part I: The Easy Way</h3>  
+                </div>
+                <div class="footer">
+                        <div class="inline">
+                          <label class="girl">
+                             <img src="/image/download.jpg" id="log" alt="logo">
```
Comment (Redvanisation) [inline index.html]: And here too

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello team  🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. The icons are not showing up, please see my comments below on how to fix that.

2. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

**Comment by Tresor11** [comment]
we just changed the image path as specified

**Review by KaushikShivam — CHANGES_REQUESTED** [review]
# Changes Required 🔴 

Hello, guys! 👋 
Awesome job on building your very first project!! Well done!

However, there are a couple of things that are missing that I'd like you to do for the review to pass.

## Mandatory changes
1. Sidebar recommended videos
This is important to fix. You're not supposed to take screenshots and use them in your code. Instead, you should
    - [ ] embed the recommended video's thumbnail image using proper tags
    - [ ] add a title of the video using proper tags
    - [ ] add the number of views paragraph using a proper tag

2. Use of semantic tag
    - [ ] Use the ```<main>``` tag for your dominant content inside the ```body``` tag instead of a ```div```.
3. Live version
     - [ ] Please add a live version of the website and include it in your README.
4. Descriptive README
    As I see, you've added little info to your README. However, you could make use of markdown to give more structure to the file. You can use this [tutorial](https://guides.github.com/features/mastering-markdown/) for reference. Things you can consider adding to your readme:
     - [ ] Author names and contact info
     - [ ] Live version of the website
     - [ ] Reference link of the youtube page and project guidelines

### That's all, guys! 
**Make these changes and submit for a code review, again**
This is your first project and you guys have done exceptionally well. I'm sure these changes won't take you much time. 👍 

**If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.**

Happy coding! 😄 
[Shivam Kaushik](https://github.com/KaushikShivam)
[Github](https://github.com/KaushikShivam) [Twitter](https://twitter.com/kShivamDev) [Linkedin](https://www.linkedin.com/in/shivam-kaushik-bb8162102/)

**Comment by Tresor11** [comment]
we corrected everything :+1:

**Comment by AdnanAfsari** [comment]
**Status: Approved**

Nice, Its a 👍.

Happy Coding

**Pull request merged**